api/nodes_test.go

```diff
@@ -153,6 +153,27 @@ func TestNodes_Info(t *testing.T) {
 	must.GreaterEq(t, 1, len(result.Events))
 }
 
+func TestNode_Stats(t *testing.T) {
+	testutil.Parallel(t)
+
+	c, s := makeClient(t, nil, func(c *testutil.TestServerConfig) {
+		c.DevMode = true
+	})
+	defer s.Stop()
+	nodesAPI := c.Nodes()
```

Comment:
ah this is one of those little "node" vs "client" battlegrounds, eh?

Reply:
I was more pointing out that it hits the `/client` API, not `/node`!  looks like there's no `client.go` to have something like `c.Clients()` presumably to avoid the name collision of "Nomad client" and "Nomad API client" ... so the Nodes{} (go) API on the (go) API Client{} is used to hit both the /nodes (http) API and the /client (http) API :sweat_smile:

Reply:
Ah! Yeah, totally.